### ISSUE TITLE
Gtk not working on m1 mac on Monterrey

### ISSUE THREAD

**Issue by gbaraldi** [JuliaGraphics/Gtk.jl#590]
The Gtk windows open but they are black, I'm not sure if this is an upstream bug or a Gtk.jl bug.
The tests give this warning : 
`(<unknown>:2879): Gtk-WARNING **: 11:59:01.081: Could not load a pixbuf from /org/gtk/libgtk/theme/Adwaita/assets/check-symbolic.svg.
This may indicate that pixbuf loaders or the mime database could not be found.
file info contains: 13
`
![Kapture 2021-11-01 at 11 59 07](https://user-images.githubusercontent.com/28694980/139693107-5a8620a0-3e97-4e93-9798-0cc4af9a8724.gif)
```julia
julia> versioninfo()
Julia Version 1.7.0-rc2
Commit f23fc0d27a (2021-10-20 12:45 UTC)
Platform Info:
  OS: macOS (arm64-apple-darwin20.6.0)
  CPU: Apple M1
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.1 (ORCJIT, cyclone)
Environment:
  JULIA_NUM_THREADS = 4
  JULIA_NUM_PRECOMPILE_TASKS = 4

```
The same happens on master tho

edit:
It's an upstream bug with GTK. I opened an issue there https://gitlab.gnome.org/GNOME/gtk/-/issues/4395

**Comment by IanButterworth** [comment]
@giordano I guess getting the Gtk jll stack to work on M1 took some work on your part? Did you see black screens then?

**Comment by giordano** [comment]
Yes, it works fine in Big Sur
<img width="1193" alt="image" src="https://user-images.githubusercontent.com/765740/139744498-3e774830-ee02-4038-88d7-c1d2800f6a02.png">

**Comment by gbaraldi** [comment]
It's even weirder, following the Gtk docs example with the button, the button does work. It just doesn't render an image for some reason.

**Comment by giordano** [comment]
This has been fixed by upgrading to v3.24.31 in https://github.com/JuliaPackaging/Yggdrasil/pull/4150, as confirmed by https://github.com/JuliaLang/julia/pull/43629#issuecomment-1003730661
